package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "soap",
-  "version": "0.16.0",
+  "name": "soap-noendlessloop",
```

Comment:
please revert your changes in `package.json` altogether in order to get your PR merged :+1: